src/client/components/heroImage/heroImage.js

```diff
@@ -0,0 +1,20 @@
+import React from 'react';
+import './heroImage.css';
+import heroImg from '../../assets/images/hero-image.png';
+import PropTypes from 'prop-types';
+
+export default function HeroImage({ heroText }) {
+  return (
+    <div className="hero-image">
+      <img src={heroImg} alt="" />
```

Comment:
Its always good idea to provide some alternative name/text in the alt attribute of the img tag.